scopes/explorer/ui/base-component-card/details/details.module.scss

```diff
@@ -0,0 +1,39 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 16px;
+  height: 96px;
+}
+.nameSpace {
+  color: var(--text-muted, #6c707c);
+  font-size: 10px;
+  font-weight: bold;
+  text-transform: uppercase;
+  margin-bottom: 7px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+.name {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  margin-bottom: 6px;
+  > span:first-child {
+    font-weight: bold;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    margin-right: 5px;
+  }
+}
+.description {
+  color: var(--text-muted, #6c707c);
+  font-size: var(--p-xs, 14px);
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
```

Comment:
check in other browser

Reply:
use the `line-clamp` property, or add all the other prefixes

Reply:
weirdly it works on safari and firefox as well without any extra prefixes